.github/workflows/nightly.yml

```diff
@@ -0,0 +1,57 @@
+name: nightly
+on:
+  # Triggers the workflow on push or pull request events but only for the
```

Comment:
Update this comment -- it's running on a schedule and not on pull request anymore.